### ISSUE TITLE
Add Plain Language - Flesch Kincaid, etc. Evaluation

### ISSUE THREAD

**Issue by mgifford** [microsoft/accessibility-insights-web#4265]
## Is your feature request related to a problem? Please describe.
Plain Language is an important for accessibility.

## Describe the desired outcome
 Would be nice to have basic Flesch Kincaid evaluation for English built-in with the Fast Pass. There are other options for other languages, but adding it for English would be nice. 

## Describe alternatives you've considered
You can run various tools independently. I have some here https://github.com/accessibility/multilingual
https://hemingwayapp.com/ is a good one.

**Comment by ferBonnin** [comment]
thanks for the feature request @mgifford! After reviewing with the team, we have decided to wait until this is a WCAG guideline/ S.C (looks like 3.0 _clear words_ is related to this).

**Comment by mgifford** [comment]
I was hoping a simple adoption of something like https://github.com/dana-ross/flesch-kincaid or even https://github.com/retextjs/retext-readability

The math isn't that complicated. I would want to see it is done for English only, as other languages aren't as well supported. 

I wasn't familiar with [WCAG 3.0's Clear Words](https://www.w3.org/WAI/GL/WCAG3/2020/how-tos/clear-words/) though, so thanks. This addresses many of the non-English limitations.

I'm worried that waiting for WCAG 3.0 isn't the best plan. I think WCAG 2 & 3 will keep developing in parallel & it may be quite a long time till anyone starts defaulting to WCAG 3.0.  Who knows, but seems that there may be some simple ways to check for language complexity that could be included earlier if there was interest.

Obviously, someone could contribute this code for consideration if they had the resources to develop it.
